### PR TITLE
Implement `CairoRunner::check_range_check_usage()`

### PR DESCRIPTION
**Pull request by azteca1998**
# Implement `CairoRunner::check_range_check_usage()`

## Description

Implementation of `CairoRunner::check_range_check_usage()` and `BuiltinRunner::get_used_perm_range_check_units()`.
Depends on #549.

## Checklist
- [ ] Linked to Github Issue
- [x] Unit tests added
- [ ] Integration tests added.
- [x] This change requires new documentation.
  - [x] Documentation has been added/updated.

### CODE REVIEW

**Comment by azteca1998** [comment]
[CairoRunner::check_range_check_usage()](https://trello.com/c/6U3hlait/62-cairorunnercheckrangecheckusage)

**Comment by azteca1998** [comment]
[BuiltinRunner::get_used_perm_range_check_units()](https://trello.com/c/JiTpg4b7/63-builtinrunnergetusedpermrangecheckunits)

**Review by juanbono — APPROVED** [review]
LGTM

File: src/vm/runners/builtin_runner/range_check.rs [hunk 1]
```diff
@@ -121,30 +129,31 @@ impl RangeCheckBuiltinRunner {
...
             let size = cells_per_instance
                 * safe_div(&bigint!(vm.current_step), &bigint!(ratio))
                     .map_err(|_| MemoryError::InsufficientAllocatedCells)?;
             Ok((used, size))
         }
     }
 
-    pub fn get_range_check_usage(&self, memory: &Memory) -> Option<(BigInt, BigInt)> {
-        let mut rc_bounds: Option<(BigInt, BigInt)> = None;
+    pub fn get_range_check_usage(&self, memory: &Memory) -> Option<(isize, isize)> {
```
Comment (fmoletta) [inline src/vm/runners/builtin_runner/range_check.rs:140]: We could use usize here, the returned values will never be negative

File: src/vm/runners/builtin_runner/range_check.rs [hunk 2]
```diff
@@ -15,31 +15,29 @@ use crate::vm::vm_core::VirtualMachine;
...
-use super::BuiltinRunner;
-
 pub struct RangeCheckBuiltinRunner {
     ratio: u32,
     base: isize,
     stop_ptr: Option<usize>,
     pub(crate) cells_per_instance: u32,
     pub(crate) n_input_cells: u32,
-    inner_rc_bound: BigInt,
+    inner_rc_bound: isize,
```
Comment (fmoletta) [inline src/vm/runners/builtin_runner/range_check.rs:24]: This value is constant and always positive, we can use usize instead
```suggestion
    inner_rc_bound: usize,
```

File: src/vm/runners/builtin_runner/range_check.rs (hunk 1 above)
Comment (azteca1998) [reply src/vm/runners/builtin_runner/range_check.rs]: I presume they'll never overflow if casted directly into `isize`. Could you confirm that?

**Review by fmoletta — APPROVED** [review]
Nice Work!

**Pull request merged**